articles/article-122-ascii.md

```diff
@@ -0,0 +1,36 @@
+<!-- BEGIN TITLE -->
+# Every letter is a number
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+If letters are represented in computers they are stored as numbers. To encode them every letter got its own decimal number between 0 and 127. The "H", for example, is encoded as the 72.
+The string "Hello world" is stored as a sequence of numbers.
+
+```
+ H  e   l   l   o  space  w   o   r   l   d
+72 101 108 108 111   32  119 111 114 108 100
+```
+
+Since things are stored binary in computers, those numbers are translated in binary code.
+In this way are not just the capital letters encoded but also the lowercase letters, the digits 0-9 and punctuation symbols.
```

Comment:
Language suggestion: Not only the capital letters are encoded in this way, but ....

Reply:
Done.